bindata/assets/alerts/api-usage.yaml

```diff
@@ -16,7 +16,7 @@ spec:
               a successful upgrade to the next cluster version.
               Refer to `oc get apirequestcounts {{ $labels.resource }}.{{ $labels.version }}.{{ $labels.group }} -o yaml` to identify the workload.
           expr: |
-            group(apiserver_requested_deprecated_apis{removed_release="1.23"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
+            group(apiserver_requested_deprecated_apis{removed_release="1.24"}) by (group,version,resource) and (sum by(group,version,resource) (rate(apiserver_request_total{system_client!="kube-controller-manager",system_client!="cluster-policy-controller"}[4h]))) > 0
```

Comment:
Since this PR is targetting 4.11, I am wondering if we should set the next version to 1.25 now instead of waiting for post-rebase. Any thoughts?

Reply:
I think @sanchezl is the best person that can answer this question.

Reply:
Given that there are at least 7 APIs being removed in 1.25, it might be useful to start making noise about it now as long as the alerts  wouldn't cause CI failures (I don't think so).